server/modules/survey/repository/surveyRepository.js

```diff
@@ -140,18 +141,22 @@ const updateSurveyDependencyGraphs = async (surveyId, dependencyGraphs, client =
 const deleteSurvey = async (id, client = db) =>
   await client.one(`DELETE FROM survey WHERE id = $1 RETURNING id`, [id])
 
-const deleteSurveyLabel = async (id, langCode, client = db) =>
-  await deleteSurveyProp(id, ['labels', langCode], client)
+const deleteSurveyLabelsAndDescriptions = async (id, langCodes, client = db) => {
+  const propsToDelete = [NodeDef.propKeys.labels, NodeDef.propKeys.descriptions]
+  const propsUpdateCond = R.pipe(
+    R.map(langCode => R.map(propName => `#- '{${propName},${langCode}}'`)(propsToDelete)),
```

Comment:
i think it's more readable if you only map on langs. what do you think?

Reply:
right